### DISCUSSION TITLE
《主页配置》文档中没有提到index.md

### DISCUSSION THREAD

**Discussion by EricTianC** [wangchucheng/hugo-eureka#42] [General]
在[主页配置文档](https://www.wangchucheng.com/zh/docs/eureka/homepage-configuration/) 中没有说明需要创建index.md文件，但在实际部署中如果没有该文件则主页配置不生效，仍显示空白。

**Comment by wangchucheng** [comment]
感谢提出该问题！我将尽快在文档中添加该内容的相关说明。

**Comment by kloudmuka** [comment]
确实文档中对这个比较重要的配置缺少描述，按照文档中创建widget后始终无法显示，后来在示例文件中才发现必须要创建index.md才能正确显示页面。

**Comment by edwardstudy** [comment]
确实。我今天也踩坑了。
